src/User/Controller/AdminController.php

```diff
@@ -346,4 +349,33 @@ public function actionForcePasswordChange($id)
         }
         $this->redirect(['index']);
     }
+
+    /**
+     * Display list session history
+     */
+    public function actionSessionHistory($id)
+    {
```

Comment:
What about checking if the feature has been enabled?

Reply:
Fix it.